apps/src/netsim/NetSimTabsComponent.js

```diff
@@ -23,10 +23,12 @@ var shouldShowTab = require('./netsimUtils').shouldShowTab;
  * @param {jQuery} rootDiv
  * @param {netsimLevelConfiguration} levelConfig
  * @param {Object} callbacks
- * @param {function} callbacks.chunkSizeChangeCallback
+ * @param {function} callbacks.chunkSizeSliderChangeCallback
```

Comment:
Updated the name of the chunk size slider callback, but didn't add an onStop callback for it because this is a strictly local property that doesn't get propagated to other clients.